doc/user/content/get-started/quickstart.md

```diff
@@ -12,299 +12,703 @@ aliases:
+      ```
+
+   1. Run a query on `flippers`.
+
+      ```mzsql
+      SELECT *
+      FROM flippers
+      WHERE flipper_id = 450;
+      ```
+
+1. In general, you may find that you rarely need to index a source. However, if
+   for some use case, you need to index a source, you can also create indexes on
+   sources, tables, and materialized views.
+
+   1. For example, run the following queries on `bids` without an index:
+
+      ```mzsql
+       SELECT * FROM bids ORDER BY bid_time DESC LIMIT 10;
+       SELECT * from bids WHERE amount > 50 LIMIT 10;
+      ```
+
+    1. Create an index on `bids`.
+
+       ```mzsql
+       CREATE INDEX bids_by_time ON bids (bid_time DESC);
+       ```
+
+    1. Run various queries on `bids`:
+
+       ```mzsql
+       SELECT * FROM bids ORDER BY bid_time DESC LIMIT 10;
+       SELECT * from bids WHERE amount > 50 LIMIT 10;
+       ```
+
+       The queries should be faster.
+
+       The above exercise is to illustrate how to create an index on
+       sources. In practice, you may find that you rarely need to index a
+       source.
+
+## Step 8. Clean up
+
+To clean up the quickstart environment:
+
+1. Use the [`DROP SOURCE ... CASCADE`](/sql/drop-source/) command to drop
+   `auction_house` source and its dependent objects, including views and indexes
+   created on the `auction_house` subsources.
+
+   ```mzsql
+   DROP SOURCE auction_house CASCADE;
+   ```
+
+1. Use the [`DROP TABLE`](/sql/drop-table) command to drop the separate
+   `known_flippers` table and its indexes.
+
+   ```mzsql
+   DROP TABLE known_flippers;
+   ```
+
+## Summary
+
+In Materialize, to keep data up-to-date, both [indexes](/concepts/indexes/) and
+[materialized views](/concepts/views/#materialized-views) <red>incrementally
+update</red> results when Materialize ingests new data. These up-to-date results
+are then immediately available and computationally free for reads.
+
+### Considerations
+
+Before creating an index, consider:
+
+- Its memory usage as well as its compute cost implications.
+
+- In Materialize, underlying data structures that are being maintained for an
+  index (or a materialized view) can be reused by other views/queries. As
+  such, an existing index may be sufficient to support your queries.
 
-As the auction house operator, you should now have a high degree of confidence that Materialize can help you implement and automate operational use cases that depend on fresh, consistent results served in an interactive way.
+- If you have created various views to act as the building blocks for a view
+  from which you will serve the results, creating an index only on the serving
+  view may be sufficient depending upon your query patterns and usage.
 
-Once you’re done exploring the auction house source, remember to clean up your environment:
+- Even without indexes, Materialize can optimize computations for queries.
+  However, this underlying work is discarded after each query run.
```

Comment:
Just leaving a note (github PRs were having an issue last night) but there will be a separate PR to mat view vs view+index ... and once that's done, in this considerations section, will add a little blurb to point users to that.